### DISCUSSION TITLE
Has anybody made Gatsby + Faust work together?

### DISCUSSION THREAD

**Discussion by cavemon** [wpengine/faustjs#1784] [Q&A]
Since migrating to Netlify, CMS previews are no longer out-of-the-box for Gatsby.  There was mention of connecting Gatsby to Faust in a [Headless WP Podcast](https://podcast.app/gatsby-with-alex-moon-e303772455), but it dates back to 2021 and is pretty vague about making it work.  

Has anybody gotten the two frameworks to work together and care to share any insight on how?

**Comment by theodesp** [comment]
Hey @cavemon thank you very much for the question. Could you ask the same question on our Discord Channel as well?

https://discord.com/channels/836253505944813629/877940107016019998

We have a few folks that have more practical Gatsby experience that we have so they might be able to give you some pointers. Thanks

**Comment by char-cole** [comment]
I'm interested in this as well. @cavemon did you find any answers on Discord that you can surface here in public?
